iota/commands/extended/get_new_addresses.py

```diff
@@ -84,12 +94,15 @@ def __init__(self):
             | f.Max(self.MAX_SECURITY_LEVEL)
             | f.Optional(default=AddressGenerator.DEFAULT_SECURITY_LEVEL),
 
+        'checksum':  f.Type(bool) | f.Optional(default=False),
+
         'seed': f.Required | Trytes(result_type=Seed),
       },
 
       allow_missing_keys = {
         'count',
         'index',
         'securityLevel',
+        'checksum',
```

Comment:
Could we keep these strings in alphabetical order, pretty please?

![pwease kitty](https://user-images.githubusercontent.com/202040/33783652-82955bfe-dcc2-11e7-969e-802bf7153a78.jpeg)